scripts/backup.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+<<note
```

Shellcheck

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)